Update sauce-connect-proxy.md (#2139)
saucelabs/sauce-docs: docs/dev/cli/sauce-connect-proxy.md

<p><small>| OPTIONAL | STRING | <span className="sauceGreen">4.7.x</span> <span className="sauceGreen">4.8.x</span> <span className="sauceGreen">4.9.x</span> | </small></p>

> **Description**: Defines external proxy auto-configuration (PAC) URI. You can input `http(s)` or `local file://URL`. Absolute paths are required when specifying a local PAC file. `--pac` supports the standard file URL format (that is, file://auth-path/local-path, see below for differences between Mac/Linux and Windows). For more information, see [Sauce Connect Proxy Setup with Additional Proxies](/secure-connections/sauce-connect/setup-configuration/additional-proxies).<br/>

Reviewer: ```suggestion
**Description**: Defines external proxy auto-configuration (PAC) URI. You can input `http(s)` or `local file://URL`. Absolute paths are required when specifying a local PAC file. `--pac` supports the standard file URL format (that is, file://auth-path/local-path). On Windows, the format looks like `file:///C:/path/to/file`. For more information, see [Sauce Connect Proxy Setup with Additional Proxies](/secure-connections/sauce-connect/setup-configuration/additional-proxies).<br/>
```

**Environment variable**: `SAUCE_PAC`<br/>
**Shorthand**: n/a

<Tabs
defaultValue="maclinux"
values={[
{label: 'Mac or Linux', value: 'maclinux'},
{label: 'Windows', value: 'windows'},
]}>

<TabItem value="maclinux">

```bash

> --pac file://Users/JohnSmith/Desktop/MyPac.pac

Reviewer 2: `--pac file:///Users/JohnSmith/Desktop/MyPac.pac`

* for local files, `auth-path` is empty

```

</TabItem>

<TabItem value="windows">

```bash

> --pac file:///Users/JohnSmith/Desktop/MyPac.pac

Reviewer 2: `--pac file:///C:/Users/JohnSmith/Desktop/MyPac.pac`

* Windows needs the drive

Reviewer: Thank you Alana!

**Environment variable**: `SAUCE_PAC`<br/>
**Shorthand**: n/a

<Tabs
defaultValue="maclinux"
values={[
{label: 'Mac or Linux', value: 'maclinux'},
{label: 'Windows', value: 'windows'},
]}>

<TabItem value="maclinux">

```bash
--pac file://Users/JohnSmith/Desktop/MyPac.pac
```

</TabItem>

<TabItem value="windows">

```bash

> --pac file:///Users/JohnSmith/Desktop/MyPac.pac

Reviewer: ```suggestion
--pac file:///C:/Users/JohnSmith/Desktop/MyPac.pac
```
